save-core/src/commonNonJsMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -55,46 +55,51 @@ class Save(
 
         reporter.beforeAll()
-        var atLeastOneExecutionProvided = false
+
         // get all toml configs in file system
-        ConfigDetector(fs)
+        val testConfigs = ConfigDetector(fs)
             .configFromFile(rootTestConfigPath)
             .getAllTestConfigsForFiles(requestedConfigs)
-            .forEach { testConfig ->
-                // iterating top-down
-                testConfig
-                    // fully process this config's configuration sections
-                    .processInPlace()
-                    // create plugins and choose only active (with test resources) ones
-                    .buildActivePlugins(requestedTests)
-                    .takeIf { plugins ->
-                        plugins.isNotEmpty() && plugins.first().isFromEnabledSuite(includeSuites, excludeSuites)
-                    }
-                    ?.also {
-                        // configuration has been already validated by this point, and if there are active plugins, then suiteName is not null
-                        reporter.onSuiteStart(testConfig.getGeneralConfig()?.suiteName!!)
-                    }
-                    ?.forEach {
-                        atLeastOneExecutionProvided = true
-                        // execute created plugins
-                        executePlugin(it, reporter)
-                    }
-                    ?.also {
-                        reporter.onSuiteEnd(testConfig.getGeneralConfig()?.suiteName!!)
-                    }
-            }
+        var atLeastOneExecutionProvided = false
+        testConfigs.forEach { testConfig ->
+            // iterating top-down
+            testConfig
+                // fully process this config's configuration sections
+                .processInPlace()
+                // create plugins and choose only active (with test resources) ones
+                .buildActivePlugins(requestedTests)
+                .takeIf { plugins ->
+                    plugins.isNotEmpty() && plugins.first().isFromEnabledSuite(includeSuites, excludeSuites)
+                }
+                ?.also {
+                    // configuration has been already validated by this point, and if there are active plugins, then suiteName is not null
+                    reporter.onSuiteStart(testConfig.getGeneralConfig()?.suiteName!!)
+                }
+                ?.forEach {
+                    atLeastOneExecutionProvided = true
+                    // execute created plugins
+                    executePlugin(it, reporter)
+                }
+                ?.also {
+                    reporter.onSuiteEnd(testConfig.getGeneralConfig()?.suiteName!!)
+                }
+        }
+        val saveToml = testConfigs.map { it.getGeneralConfig()?.configLocation }
+        val excluded = testConfigs.map { it.getGeneralConfig()?.excludedTests?.toMutableList() ?: emptyList() }
+            .reduceOrNull { acc, list -> acc + list }
```

Comment:
val excludedNote = excluded?.let { 
    Note: please check excludedTests: $excluded in following save.toml files: $saveToml
} ?: ""

Reply:
Fixed